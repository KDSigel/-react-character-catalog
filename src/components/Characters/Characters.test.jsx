import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import Characters from "./Characters";


it('should render the Character', () => {

const characters = [{
    image: 'someurl',
    name: 'fred',
    species: 'ape',
    status: 'dead',
}]

    const { container } = render(
      <MemoryRouter>
        <Characters characters={characters}/>
      </MemoryRouter>
    );
  
    expect(container).toMatchSnapshot();
  });