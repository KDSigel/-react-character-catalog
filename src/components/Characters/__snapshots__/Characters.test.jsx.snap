// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Character 1`] = `
<div>
  <ul
    aria-label="characters"
    class="list"
  >
    <li
      class="item"
    >
      <a
        href="/characters/undefined"
      >
        <figure>
          <img
            alt="fred"
            class="image"
            src="someurl"
          />
          <figcaption
            class="linkText"
          >
            fred
          </figcaption>
        </figure>
      </a>
    </li>
  </ul>
</div>
`;
